sentry-android-core/src/main/java/io/sentry/android/core/AndroidTransactionProfiler.java

```diff
@@ -41,6 +41,20 @@ final class AndroidTransactionProfiler implements ITransactionProfiler {
   private long transactionStartNanos;
   private long profileStartCpuMillis;
 
+  public AndroidTransactionProfiler(
```

Comment:
is this ctor needed?
I'm going to create an instance of the profiler without the `SentryAndroidOptions` (for startup profiling), so i'd like to avoid adding other constructors with options as params, if possible